dor/service_layer/handlers/FilesystemHandler.py

```diff
@@ -0,0 +1,16 @@
+import os
+from pathlib import Path
+from dor.providers.FileProvider import FileProvider
+
+
+class FilesystemHandler(FileProvider):
```

Comment:
Maybe `FileSystemFileProvider`? `FilesystemHandler` seems too far away from the ABC name.